pkg/koordlet/statesinformer/config.go

```diff
@@ -18,26 +18,33 @@ package statesinformer
 
 import (
 	"flag"
+	"time"
 
 	corev1 "k8s.io/api/core/v1"
 )
 
 type Config struct {
 	KubeletPreferredAddressType string
-	KubeletSyncIntervalSeconds  int
-	KubeletSyncTimeoutSeconds   int
+	KubeletSyncInterval         time.Duration
+	KubeletSyncTimeout          time.Duration
+	InsecureKubeletTLS          bool
+	KubeletReadOnlyPort         uint
```

Comment:
This field doesn't seem to be used, am I missing it?

Reply:
this PR only introduce new options. further implementation will be done in another PR.